### ISSUE TITLE
How to run?

### ISSUE THREAD

**Issue by chamow97** [pedrodiamel/kaggle-human-protein-atlas#1]
First of all, thanks for the repository. It is useful for beginners like me. It would be great if you can give steps on how to run and generate results. 
I downloaded external data using your script. then tried running train.py. I am getting errors, don't know where am I going wrong.

**Comment by pedrodiamel** [comment]
Hello chamow97 
I used the [pytorchvision](https://github.com/pedrodiamel/pytorchvision) project for data augmentation and aux functions (visualization ...). 

## Installation

> git clone https://github.com/pedrodiamel/pytorchvision.git
> cd pytorchvision
> python setup.py build_ext --inplace
> python setup.py install
> 

## Run
For run this project (kaggle-human-protein-atlas): 
> cd runs
> bash train.sh

The parameters in train.sh for our results:

> DATA=$HOME/.kaggle/competitions/human-protein-atlas-image-classification
> NAMEDATASET='atlas'
> PROJECT='../out/netruns'
> EPOCHS=1000
> BATCHSIZE=128 #128, 32
> LEARNING_RATE=0.0001  # 0.1, 0.03, 0001, 00001
> MOMENTUM=0.9
> PRINT_FREQ=100
> WORKERS=32 #10 or 1
> RESUME='chk000255.pth.tar' #model_best, chk000000
> GPU=0
> ARCH='resnet18' # resnet18, resnet34, resnet50, preactresnet18, inception_v4, nasnetalarge
> LOSS='mix' #focal, bcewl, mix
> OPT='adam' #sgd, adam
> SCHEDULER='step'
> SNAPSHOT=5
> NUMCLASS=28
> NUMCHANNELS=3 #4
> IMAGESIZE=512 #1024, 512, 331
> EXP_NAME='atlas_baseline_'$ARCH'_'$LOSS'_'$OPT'_'$NAMEDATASET'_000' 

I hope to help you to run this project.

**Comment by chamow97** [comment]
Thank you @pedrodiamel. Great work!
